GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -152,20 +149,17 @@ class ChatViewModel: EngagementViewModel, ViewModel {
     override func start() {
         super.start()
 
-        loadHistory { [weak self] in
-            // At this point we need to be sure that CoreSDK is configured.
-            // This will restore engagement if one was not completed earlier.
-            // Otherwise in case of ongoing engagement, visitor will not receive
-            // messages from operator until message is sent from visitor.
-            self?.interactor.withConfiguration { [weak self] in
-                guard let self = self else { return }
-                // We only proceed to considering enqueue flow if `startAction` is about starting of engagement.
-                guard case .startEngagement = self.startAction else { return }
-                // We enqueue eagerly in case if this is the first engagement for visitor (by  evaluating previous chat history)
-                // or in case if engagement has been restored.
-                if self.chatStorageState().isEmpty() || self.environment.getCurrentEngagement() != nil {
-                    self.enqueue(mediaType: .text)
-                }
+        loadHistory { [weak self] history in
```

Comment:
Does `loadHistory` work for ongoing engagement?

Reply:
yes, because `withConfiguration` happens early, so it doesn't require to do it again here.